test/Microsoft.ML.TestFramework/Datasets.cs

```diff
@@ -163,8 +163,8 @@ public static class TestDatasets
         public static TestDataset WikiDetox = new TestDataset
         {
             name = "WikiDetox",
-            trainFilename = "Input/WikiDetoxAnnotated160kRows.tsv",
-            testFilename = "Input/WikiDetoxAnnotated160kRows.tsv"
+            trainFilename = "external/WikiDetoxAnnotated160kRows.tsv",
```

Comment:
I believe the casing here is different than what we use in the `build.proj`.

Here it is:
`WikiDetoxAnnotated160kRows.tsv`
In build.proj it is:
`wikiDetoxAnnotated160kRows.tsv`

When running on a case sensitive file system (i.e. Linux), they need to be the same.